hyperstart.go

```diff
@@ -334,14 +373,57 @@ func (h *hyper) startContainer(pod Pod, contConfig ContainerConfig) error {
 		return err
 	}
 
+	err = h.exec(pod, contConfig, contConfig.Cmd)
```

Comment:
Let me try to understand here: When starting an hyperstart container, we're still sending the `NEWCONTAINER` command to hyperstart but now we're also exec'ing a `pause-podID-containerID` ?
I don't think we need this at all, but please prove me wrong.